### DISCUSSION TITLE
Ford Sync 1.0: Compatibility? Alternative SDKs?

### DISCUSSION THREAD

**Discussion by charliecrawford2005** [ianjwhite99/sync-connect-sdk#77] [General]
Hello I was curious if anyone was aware of an SDK that is compatible with the Ford Sync 1.0 (embedded windows for automotive) system that would allow me to push code to my Sync? I would like to develop some basic applications for my 2010 Ford f-150. It has the most basic version of the Sync interface offered by ford at the time (the one with the LCD display in the center). I guess I am kind of starting from scratch in terms of figuring out what tools I need and where to find them. I have some software experience, but none with car coding. 

Let me know if there is a better place to reach out for information.

Thank you.

**Comment by HKLM** [comment]
Your options are extremely limited, might not even be possible with sync 1. But you can interact with the OBD2 interface and use it. I would suggest to check out the [OpenXC project](http://openxcplatform.com/) website / [OpenXC github page](https://github.com/openxc) Ford is one of the founding members of OpenXC.

**Reply by charliecrawford2005** [reply]
Thank you very much for the reply, I will check out those resources.
